subprojects/gradle/build-trace/src/main/kotlin/com/avito/android/build_trace/internal/critical_path/CriticalPathListener.kt

```diff
@@ -1,98 +0,0 @@
-package com.avito.android.build_trace.internal.critical_path
-
-import com.avito.android.build_trace.extensionName
-import com.avito.android.build_trace.internal.TaskDependenciesResolutionResult
-import com.avito.android.build_trace.internal.predecessors
-import com.avito.android.build_trace.internal.type
-import com.avito.android.gradle.metric.AbstractBuildEventsListener
-import com.avito.android.gradle.profile.BuildProfile
-import com.avito.android.gradle.profile.TaskExecution
-import com.avito.composite_exception.CompositeException
-import com.avito.graph.OperationsPath
-import com.avito.graph.ShortestPath
-import com.avito.logger.GradleLoggerFactory
-import com.avito.logger.create
-import org.gradle.BuildResult
-import org.gradle.api.Task
-import java.io.PrintWriter
-import java.io.StringWriter
-
-internal class CriticalPathListener(
```

Comment:
Move to another module: https://github.com/avito-tech/avito-android/pull/1038/files#r643787575